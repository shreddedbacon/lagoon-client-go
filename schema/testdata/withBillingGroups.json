{
  "data": {
    "projectByName": {
      "id": 18,
      "name": "high-cotton",
      "autoIdle": 1,
      "branches": "true",
      "pullrequests": "true",
      "privateKey": "-----BEGIN OPENSSH PRIVATE KEY-----\nb3BlbnNzaC1rZXktdjEAAAAABG5vbmUAAAAEbm9uZQAAAAAAAAABAAAAMwAAAAtzc2gtZW\nQyNTUxOQAAACCiXQRY2swIZ5cMS1pBd/FMeZ65EnXUAhyxIXKB8FtaoQAAAIixOPe0sTj3\ntAAAAAtzc2gtZWQyNTUxOQAAACCiXQRY2swIZ5cMS1pBd/FMeZ65EnXUAhyxIXKB8FtaoQ\nAAAEBpG78n7RgfyHGi2suzBxFDLCaZ3TfHI/s60JGzxh6dFaJdBFjazAhnlwxLWkF38Ux5\nnrkSddQCHLEhcoHwW1qhAAAAAAECAwQF\n-----END OPENSSH PRIVATE KEY-----\n",
      "productionEnvironment": "Master",
      "activeSystemsDeploy": "lagoon_openshiftBuildDeploy",
      "activeSystemsTask": "lagoon_openshiftJob",
      "activeSystemsRemove": "lagoon_openshiftRemove",
      "activeSystemsPromote": "lagoon_openshiftBuildDeploy",
      "storageCalc": 1,
      "openshiftProjectPattern": null,
      "developmentEnvironmentsLimit": 5,
      "gitUrl": "ssh://git@192.168.42.1:2222/git/project18.git",
      "groups": [
        {
          "__typename": "BillingGroup",
          "name": "High Cotton Billing Group",
          "currency": "USD",
          "billingSoftware": "null"
        },
        {
          "__typename": "Group",
          "name": "project-high-cotton",
          "members": [
            {
              "user": {
                "email": "default-user@high-cotton",
                "sshKeys": [
                  {
                    "name": "auto-add via api",
                    "created": "2020-02-11 09:00:42",
                    "keyFingerprint": "",
                    "keyType": "ssh-ed25519",
                    "keyValue": "AAAAC3NzaC1lZDI1NTE5AAAAIKJdBFjazAhnlwxLWkF38Ux5nrkSddQCHLEhcoHwW1qh"
                  }
                ],
                "firstName": null,
                "lastName": null
              },
              "role": "MAINTAINER"
            }
          ]
        },
        {
          "__typename": "Group",
          "name": "ui-customer",
          "members": []
        }
      ],
      "notifications": [],
      "openshift": {
        "id": 2
      },
      "envVariables": [],
      "environments": [
        {
          "name": "Master",
          "deployType": "branch",
          "deployBaseRef": "Master",
          "deployHeadRef": null,
          "deployTitle": null,
          "environmentType": "production",
          "openshiftProjectName": "high-cotton-master",
          "autoIdle": 1,
          "envVariables": []
        },
        {
          "name": "Staging",
          "deployType": "branch",
          "deployBaseRef": "Staging",
          "deployHeadRef": null,
          "deployTitle": null,
          "environmentType": "development",
          "openshiftProjectName": "high-cotton-staging",
          "autoIdle": 1,
          "envVariables": []
        },
        {
          "name": "Development",
          "deployType": "branch",
          "deployBaseRef": "Development",
          "deployHeadRef": null,
          "deployTitle": null,
          "environmentType": "development",
          "openshiftProjectName": "high-cotton-development",
          "autoIdle": 1,
          "envVariables": []
        },
        {
          "name": "PR-175",
          "deployType": "pullrequest",
          "deployBaseRef": "target",
          "deployHeadRef": "source",
          "deployTitle": "PR-175",
          "environmentType": "development",
          "openshiftProjectName": "high-cotton-pr-175",
          "autoIdle": 1,
          "envVariables": []
        }
      ]
    }
  }
}
