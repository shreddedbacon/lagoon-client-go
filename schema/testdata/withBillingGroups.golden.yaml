billingGroups:
- billingSoftware: "null"
  currency: USD
  name: High Cotton Billing Group
groups:
- name: ui-customer
projects:
- autoIdle: 1
  billingGroups:
  - High Cotton Billing Group
  environments:
  - autoIdle: 1
    deployBaseRef: Master
    deployType: branch
    environmentType: production
    name: Master
    openshiftProjectName: high-cotton-master
  - autoIdle: 1
    deployBaseRef: Staging
    deployType: branch
    environmentType: development
    name: Staging
    openshiftProjectName: high-cotton-staging
  - autoIdle: 1
    deployBaseRef: Development
    deployType: branch
    environmentType: development
    name: Development
    openshiftProjectName: high-cotton-development
  - autoIdle: 1
    deployBaseRef: target
    deployHeadRef: source
    deployTitle: PR-175
    deployType: pullrequest
    environmentType: development
    name: PR-175
    openshiftProjectName: high-cotton-pr-175
  gitUrl: ssh://git@192.168.42.1:2222/git/project18.git
  groups:
  - ui-customer
  name: high-cotton
  productionEnvironment: Master
  storageCalc: 1
